index.html

```diff
@@ -2,30 +2,49 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width">
+    <meta name="viewport"content="width=device-width">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Portfolio</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,400&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500&family=Roboto:ital,wght@0,500;1,400&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="css/style.css">
     <script src="https://kit.fontawesome.com/4e96f49cc5.js" crossorigin="anonymous"></script>
+    
+    <title>Portfolio</title>
+
 </head>
 <body>
     <header>
         <h1 class="portfolio">Portfolio</h1>
         <nav>
-            <ul class="desktop-nav">
-                <li class="nav-item" ><a>Home</a></li>
-                <li class="nav-item" ><a>About</a></li>
-                <li class="nav-item" ><a>Contact</a></li>
-            </ul>
 
-            <div class="mobile-nav">
+            <div class="hamburger-menue" id="mobileMenu">
                 <div class="line"></div>
                 <div class="line"></div>
                 <div class="line"></div>
             </div>
+            
+            <ul class="desktop-nav">
+                <li class="nav-item" ><a herf="#Portfolio">Portfolio</a></li>
+                <li class="nav-item" ><a herf="#About">About</a></li>
+                <li class="nav-item" ><a herf="#Contact">Contact</a></li>
+            </ul>
+
+            <div class="mobile-nav" id="mobile-nav">
+                <ul class="mobile-nav-content">
+                    <li class="close fa-solid fa-x fa-xs"></li>
+                    <li class="nav-item" ><a herf="#Portfolio">Portfolio</a></li>
+                    <li class="nav-item" ><a herf="#About">About</a></li>
+                    <li class="nav-item" ><a herf="#Contact">Contact</a></li>
+                </ul>
+            </div>
+
```

Comment:
- [ ] I think here you need to blur the background to match the Figma design as much as possible ✅✅

- [ ] Kindly use the same font weight as the Figma template ✅💪🏻

- [ ] Please also as I can see here the close button width in sont the same width in the Figma so Kindly Fix it to be the same 🔥🔥


- Your design
![image](https://user-images.githubusercontent.com/58553711/179653253-57fd2998-3d1d-4621-ae44-6d6e4665c73d.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/179653320-8a269381-8b5f-4ad3-9d92-608cb64e477e.png)

Reply:
solved, thanks i appreciate the detailed and concise feedback